geode-core/src/main/java/org/apache/geode/distributed/ConfigurationProperties.java

```diff
@@ -1921,46 +1921,48 @@
    */
   String OFF_HEAP_MEMORY_SIZE = "off-heap-memory-size";
   /**
-   * The static String definition of the <i>"redis-bind-address"</i> property <a
-   * name="redis-bind-address"/a>
+   * The static String definition of the <i>"compatible-with-redis-bind-address"</i> property <a
+   * name="compatible-with-redis-bind-address"/a>
    * </p>
-   * <U>Description</U>: Specifies the address on which the Redis API for Geode is listening. If set
-   * to the empty string or this property is not specified, the server listens on all local
-   * addresses.
+   * <U>Description</U>: Specifies the address on which Geode's API for Redis Compatibility is
+   * listening. If set to the empty string or this property is not specified, the server listens on
+   * all local addresses.
    * </p>
    * <U>Default</U>: ""
    */
-  String REDIS_BIND_ADDRESS = "redis-bind-address";
+  String REDIS_BIND_ADDRESS = "compatible-with-redis-bind-address";
   /**
-   * The static String definition of the <i>"redis-enabled"</i> property <a
-   * name="redis-enabled"/a>
+   * The static String definition of the <i>"compatible-with-redis-enabled"</i> property <a
+   * name="compatible-with-redis-enabled"/a>
    * </p>
-   * <U>Description</U>: When the default value of false, the Redis API for Geode is not available.
-   * Set to true to enable the Redis API for Geode.</td>
+   * <U>Description</U>: When the default value of false, the Geode's API for Redis compatibility is
+   * not available.
+   * Set to true to enable the Geode's API for Redis compatibility.</td>
    * </p>
    * <U>Default</U>: false
-   * <td>redis-enabled</td>
-   * <td>When the default value of false, the Redis API for <%=vars.product_name%> is not available.
-   * Set
-   * to true to enable the Redis API for <%=vars.product_name%>.</td>
+   * <td>compatible-with-redis-enabled</td>
+   * <td>When the default value of false, the <%=vars.product_name%> API for compatibility with
+   * Redis is not available.
+   * Set to true to enable the <%=vars.product_name%> API for compatibility with Redis.</td>
```

Comment:
I recognize that this code was here beforehand, but I don't think we can use the `<%=vars.product_name%>` variable here. Not sure that it gets resolved.  Also, it doesn't seem like this is a table, all the other descriptions utilize a <p> tag.

Reply:
I left everything else the same and just changed the text, so as far as I know it seems like it should be fine as is

Reply:
I don't think it's ok that it's there or that it will get resolved.  I'm not sure how it got there beforehand (it might've actually been me who put it in), but we should fix it.

Reply:
fixed